soccer/main.py

```diff
@@ -14,16 +15,45 @@
 LEAGUE_IDS = leagueids.LEAGUE_IDS
 TEAM_NAMES = teamnames.team_names
 
+
+def write_config_file():
+    """Write API key to config.py file"""
+    print("No API key found!")
+    print("Please visit {0} and get an API token.".format(BASE_URL))
+    while True:
+        confkey = raw_input("Please enter API key: ")
+        if len(confkey) == 32:  # 32 chars
+            try:
+                int(confkey, 16)  # hexadecimal
+            except ValueError:
+                print("Invalid API key, try again")
+            else:
+                break
+        else:
+            print("Invalid API key, try again")
+    here = os.path.dirname(os.path.abspath(__file__))
+    with open(os.path.join(here, "config.py"), "wb") as conffile:
+        conffile.write('config = {\r\n')
+        conffile.write('    "SOCCER_CLI_API_TOKEN": "{0}",\r\n'.format(confkey))
+        conffile.write('}\r\n')
+    print("Config file written! Please restart Soccer CLI.")
+    sys.exit(2)
+
```

Comment:
i personally do not like this solution....we are doing too much for the user. It should be sufficient to let user know that api token was not found. There are currently two main ways to read api tokens: environment variable and reading from a config.py file. But reading from a config.py is mainly for development purposes. For those only concerned about only using the CLI, the environment variable is the only way to read the token. We can give the user an alternate option where they can pass API via the command line